packages/filebrowser/src/listing.ts

```diff
@@ -808,6 +808,15 @@ export class DirListing extends Widget {
       // next value).
       checkAllCheckbox.dataset.checked = String(allSelected);
       checkAllCheckbox.dataset.indeterminate = String(someSelected);
+
+      const trans = this.translator.load('jupyterlab');
+      checkAllCheckbox?.setAttribute(
+        'aria-label',
+        trans.__(
+          '%1 all files and directories',
+          allSelected || someSelected ? 'Unselect' : 'Select'
+        )
```

Comment:
'Unselect' and 'Select' are missing translation.


I think that when possible it is always preferable to give translators more control. "Select all" may be a single word or subject to declension different than "Deselect all". I believe that variable substitution should be only used for numbers and arbitrary inputs (e.g. file names).

```suggestion
        allSelected || someSelected
            ? trans.__('Deselect all files and directories')
            : trans.__('Select all files and directories')
        )
```

Reply:
Thanks @krassowski, I thought the variables were translated using variable substitution.